src/fabric.c

```diff
@@ -344,6 +344,8 @@ void fi_ini(void)
 {
 	char *param_val = NULL;
 
+	OFI_OSD_INIT();
```

Comment:
one of the reason why it is here is initialize mutex (critical section) ini_lock prior of first usage. this call could not be moved down